ingestion/src/metadata/ingestion/source/sql_source.py

```diff
@@ -200,7 +200,7 @@ def get_databases(self) -> Iterable[Inspector]:
         yield inspect(self.engine)
 
     def get_table_fqn(self, service_name, schema, table_name) -> str:
-        return f"{service_name}.{schema}.{table_name}"
+        return f"{service_name}.{schema}.{table_name}".lower()
```

Comment:
Hey @troyel, can you move this lower to line 436 - as this method is being used in multiple places and can break few things.